sympy/stats/crv_types.py

```diff
@@ -2603,6 +2719,19 @@ def UniformSum(name, n):
     --------------------------------
                 (n - 1)!
 
+    >>> cdf(X)(z)
+    Piecewise((0, z < 0), (Sum((-1)**_k*(-_k + z)**n*binomial(n, _k),
+                            (_k, 0, floor(z)))/factorial(n), n >= z), (1, True))
+
+
+    Compute cdf with specific 'x' and 'n' values as follows :
+    >>> cdf(UniformSum("x", 5), evaluate=False)(2).doit()
+    9/40
+
+    In above formof cdf functtion evaluate=False prevents an attempt at premature evaluation of the sum,
```

Comment:
"formof" is  a typo which I haven't noticed earlier. The second sentence seems unnecessary: we see it was evaluated. This should be shorter. I suggest:

The argument evaluate=False prevents an attempt at evaluation of the sum for general n, before the argument 2 is passed.

Also, docstrings should be wrapped at 72 characters, if I remember correctly (from your previous PR)